src/Models/ProductOptionTypeValue.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Rapidez\Core\Models;
+
+use Illuminate\Database\Eloquent\Casts\Attribute;
+
+class ProductOptionTypeValue extends Model
+{
+    protected $table = 'catalog_product_option_type_value';
+
+    protected $primaryKey = 'option_type_id';
+
+    public $timestamps = false;
+
+    protected $appends = ['title', 'price', 'price_label'];
+
+    protected $with = ['titles', 'prices'];
+
+    public function option()
+    {
+        return $this->belongsTo(config('rapidez.models.product_option'), 'option_id');
+    }
+
+    protected function title(): Attribute
+    {
+        return Attribute::make(
+            get: function () {
+                $titles = $this->titles->pluck('title', 'store_id')->toArray();
+
+                return $titles[config('rapidez.store')] ?? $titles[0];
```

Comment:
I think you could keep this as a collection and then get the first value using `$titles->first()`

Reply:
Yea, doing that with `price` as we need multiple columns there but with the title, we just need the title for the current store. Besides the ID's there is nothing else in the table so that's why I choose to return the title directly.

Reply:
But.. I've created a new macro for this to make it cleaner: https://github.com/rapidez/core/pull/282/commits/9cb70b1933b4eb1d1969a96a0e26444a199c7970